internal/pipeline/testdata/app-with-db/configs/Kptfile

```diff
@@ -0,0 +1,19 @@
+apiVersion: kpt.dev/v1alpha2
+kind: Kptfile
+metadata:
+  name: app-with-db
+pipeline:
+  mutators:
+  - image: gcr.io/kpt-fn/set-namespace:unstable
+    configMap:
+      namespace: staging
+  - image: gcr.io/kpt-fn/set-label:unstable
+    config:
```

Comment:
this should be a configMap for brevity

Reply:
@Shell32-Natsu is addressing this, will use the `configMap` version once the PR is in.

Reply:
We have already updated to support ConfigMap. @mengqiy Have we released them?

Reply:
Yes, we have released them. Tag `unstable` is always from HEAD.

Reply:
Thanks guys! Using `configMap` now, just verified with the latest `set-label` and it works.